resources/assets/components/InboxItem/index.js

```diff
@@ -52,12 +70,12 @@ class InboxItem extends React.Component {
           </ul>
         </div>
         <div className="container__block -third">
-          {post['user'] ?
+          {post['user'] || signup.user ?
             <div>
-              <h2>{post['user']['first_name']} {post['user']['last_name']}, {calculateAge(post['user']['birthdate'])}</h2>
+              <h2>{first_name} {last_name}, {birthdate}</h2>
```

Comment:
ahh I see - so cool! For the [second example](https://github.com/DoSomething/rogue/blob/ac76c91a7607afd55adee9f523270ee3c3cc9561/app/Http/Controllers/CampaignsController.php#L108), we're not actually passing `$signups` anymore since this PR grabs it via API call. Is it ok to do for the first (in the `showInbox` function and then when rendering the user information, doing a conditional like `if (displayName)` use that info and if not get the user info another way?